index.html

```diff
@@ -5,9 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
 </head>
 <body>
-    <header></header>
-    <section></section>
+    <header>
+        <div class="status-bar"></div>
+        <div class="toolbar">
```

Comment:
- [ ] KIndly ensure that your logo and hamburger are properly aligned to the center of the header container to match the figma design you chose 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/198709356-cd8d0e08-11f0-43a3-b336-52b92d8c1dd4.png)  | ![image](https://user-images.githubusercontent.com/63473584/198709456-7d2b0e39-d716-4fc0-a935-f13b86e652e9.png)l  |